gen3/bin/kube-setup-gearbox-middleware.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+#
+# Deploy the metdata service.
+#
+
+source "${GEN3_HOME}/gen3/lib/utils.sh"
+gen3_load "gen3/gen3setup"
+
+gen3 roll gearbox-middleware
+g3kubectl apply -f "${GEN3_HOME}/kube/services/gearbox-middleware/gearbox-middlware-service.yaml"
+
+cat <<EOM
+The gearbox-middleware service has been deployed onto the k8s cluster.
+EOM
```

Comment:
You are missing the update of the config/secrets. Follow this as an example: https://github.com/chicagopcdc/cloud-automation/blob/pcdc_dev/gen3/bin/kube-setup-gearbox.sh